nni/compression/pytorch/utils/check_ddp.py

```diff
@@ -25,7 +36,7 @@ def check_ddp_model(model: nn.Module):
         if "gradient_as_bucket_view" in attr_dicts:
             ddp_params["gradient_as_bucket_view"] = attr_dicts.get("gradient_as_bucket_view", False)
         # when torch version <= 1.10.0, there is no param "static_graph"
-        if "static_graph" in attr_dicts:
+        if "static_graph" in attr_dicts and torch_version not in ["1.9.0", "1.9.1", "1.10.0"]:
```

Comment:
suggestion: use `torch.__version__ <= '1.10.0'` directly

Reply:
good suggestion, I will modify it.